.github/actions/start-k8s-cluster/action.yaml

```diff
@@ -3,8 +3,8 @@ description: Creates and configures a k8s cluster for all things TNF-related
 inputs:
   working_directory:
     default: .
-  k8s_cluster_version:
-    default: v0.11.1
+  kind_version:
```

Comment:
Also update the vagrant script @:
/home/deliedit/partner/config/vagrant/scripts/bootstrap-cluster.sh

Reply:
Done.